templates/todo/projects/nodejs-mongo-aks/.repo/bicep/infra/hooks/predeploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Retrieving cluster credentials"
+az aks get-credentials --resource-group ${AZURE_RESOURCE_GROUP} --name ${AZURE_AKS_CLUSTER_NAME}
+
+appConfigProviderStatus=$(helm status azureappconfiguration.kubernetesprovider -n azappconfig-system)
+
+if [ $? -eq 0 ]; then
+  echo "Azure App configuration Kubernetes provider installed, skipping"
+else
+  echo "Azure App configuration Kubernetes provider not installed, installing"
+  helm install azureappconfiguration.kubernetesprovider oci://mcr.microsoft.com/azure-app-configuration/helmchart/kubernetes-provider --namespace azappconfig-system --create-namespace
```

Comment:
Install the azureappconfiguration.kubernetesprovider in hook script since helm support for `azd` does not support OCI based helm repo.

Reply:
Issue tracked here https://github.com/Azure/azure-dev/issues/3562